astropy/stats/src/compute_bounds.c

```diff
@@ -43,7 +43,7 @@ void compute_sigma_clipped_bounds(double data_buffer[], int count, int use_media
 
       std = 0;
       for (i = 0; i < count; i++) {
-        std += pow(mean - data_buffer[i], 2);
+        std += (data_buffer[i] - mean) * (data_buffer[i] - mean);
```

Comment:
Would it be more performant to assign `data_buffer[i] - mean` to a variable (e.g., `x`) and using `x * x`?

Reply:
Unlikely to be the case, since the compiler should put the result of the subtraction into a register and then multiply the contents of the register and that's effectively the same as saving into a separate variable (with local scope).